src/third-party/PhishingController.ts

```diff
@@ -139,7 +139,7 @@ export class PhishingController extends BaseController<PhishingConfig, PhishingS
   }
 
   private async queryConfig(input: RequestInfo): Promise<EthPhishingResponse | null> {
-    const response = await fetch(input);
+    const response = await fetch(input, {cache: "reload"});
```

Comment:
```suggestion
    const response = await fetch(input, {cache: "no-cache"});
```
We believe `no-cache` is the default, but are happy to make it explicit. We prefer no-cache to reload because it avoids re-fetching the entire file, and only requires the hash.